run_dialyzer.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -eu
+
+#make dialyzer
+DIALYZER_ANALYSE_APP=emqx_auth_lwm2m make dialyzer
```

Comment:
Please use comment as an example for `DIALYZER_ANALYSE_APP` but do not specify `emqx_auth_lwm2m` here.